.github/workflows/test-desktop.yml

```diff
@@ -413,26 +413,3 @@ jobs:
           name: summary.json
           path: ${{ github.workspace }}/summary.json
 
```

Comment:
Don't we want to remove this part too?
```
      - name: Upload Allure Report
        if: ${{ !cancelled() }}
        uses: actions/upload-artifact@v3
        with:
          name: allure-results-linux
          path: apps/ledger-live-desktop/allure-results
```

Reply:
This was actually what I did first and then changed my mind to only focus on the failing job: Allure Upload to Server
This way, we are also keeping the generation of Allure reports and ability to download them from the job page (ex: https://github.com/LedgerHQ/ledger-live/actions/runs/7813019200)
Ok to change it though.